plugins/modules/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -312,6 +312,8 @@
     port_channel="uni/infra/funcprof/accbundle-{0}",
     switch_port="uni/infra/funcprof/accportgrp-{0}",
     vpc="uni/infra/funcprof/accbundle-{0}",
+    fex_port_channel="uni/infra/funcprof/accbundle-{0}",
+    fex_vpc="uni/infra/funcprof/accbundle-{0}",
```

Comment:
Is the mapping accurate?

Reply:
Yes, port_channel, fex_port_channel and fex_vpc all are using the same type of DN.

Reply:
as they use the same DN, can we define a string variable == "uni/infra/funcprof/accbundle-{0}" and use it in line 312-316.